examples/stable-diffusion/image_to_video_generation.py

```diff
@@ -188,41 +230,68 @@ def main():
     if args.bf16:
         kwargs["torch_dtype"] = torch.bfloat16
 
```

Comment:
```suggestion
    # Set seed before running the model
    set_seed(args.seed)
```

personal suggestion!

Reply:
Done, Thanks.